manifests/Jimmys-Ship-Emporium.yaml

```diff
@@ -0,0 +1,13 @@
+name: Jimmys-Ship-Emporium
+authors: Jimmy Firyh
+homepage: https://github.com/JimmyFiryh/Jimmys-Ship-Emporium
+license: CC-BY-SA-4.0
```

Comment:
This is incoherent - your `LICENSE` file states GPL3, and your `copyright` file mentions no license at all.